### PR TITLE
refactor: add toggleable Traveller info on ticket purchase 

### PR DESCRIPTION
**Pull request by monopoint**
closes https://github.com/AtB-AS/kundevendt/issues/13

### CODE REVIEW

File: src/screens/Ticketing/Purchase/Overview/components/TravellerSelection.tsx [hunk 1]
```diff
@@ -48,16 +54,51 @@ export default function TravellerSelection({
...
+
+        <View
+          style={{
+            flex: 1,
+            flexDirection: 'row',
+            flexGrow: 1,
+            justifyContent: 'flex-end',
+          }}
+        >
+          <View></View>
```
Comment (Nisha1306) [inline src/screens/Ticketing/Purchase/Overview/components/TravellerSelection.tsx:78]: redundant?

File: src/translations/screens/subscreens/TicketTraveller.ts [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+function specificOverrides(
+  travellerType: string,
+  ticketType: PreassignedFareProductType | undefined,
+) {
+  if (ticketType == undefined) return false;
+
+  switch (ticketType) {
+    case 'single':
+      switch (travellerType) {
+        case TravellerType.child:
```
Comment (Nisha1306) [inline src/translations/screens/subscreens/TicketTraveller.ts:21]: Is there any specific reason we have only child for single ticket type here?

Comment (Nisha1306) [reply src/translations/screens/subscreens/TicketTraveller.ts:21]: ohhkk...got it. 👍

**Review by Nisha1306 — APPROVED** [review]

File: src/screens/Ticketing/Purchase/Overview/components/TravellerSelection.tsx (hunk 1 above)
Comment (monopoint) [reply src/screens/Ticketing/Purchase/Overview/components/TravellerSelection.tsx:78]: yes indeed!

**Pull request merged**